libafl/src/inputs/bytes.rs

```diff
@@ -108,4 +108,10 @@ impl BytesInput {
     pub fn new(bytes: Vec<u8>) -> Self {
         Self { bytes }
     }
+
+    /// Take the ownership of the inner bytes, consume self
+    #[must_use]
+    pub fn into_bytes(self) -> Vec<u8> {
```

Comment:
This could be something like
```rust
impl From<BytesInput> for Vec<u8> {
    fn from(input: BytesInput) -> Vec<u8>  {
        input.bytes
    }
}
```

Then you can usr `.into()`

Reply:
Good point, seems more rusty way

Reply:
Done